app/src/main/java/org/jellyfin/mobile/ui/utils/CenterRow.kt

```diff
@@ -10,9 +10,10 @@ import androidx.compose.ui.Modifier
 
 @Composable
 inline fun CenterRow(
+    modifier: Modifier = Modifier,
     content: @Composable RowScope.() -> Unit,
 ) = Row(
-    modifier = Modifier.fillMaxWidth(),
+    modifier = modifier.fillMaxWidth(),
```

Comment:
I'm not 100% sure about the modifier application order, but normally, you would do size first, then padding. But it's possibly that this doesn't matter if you use `fillMaxWidth()`.

Reply:
Changed to
```
 modifier = Modifier
        .fillMaxWidth()
        .then(modifier),
```

Reply:
Perfect!